x-pack/filebeat/module/threatintel/abusemalware/_meta/fields.yml

```diff
@@ -0,0 +1,49 @@
+- name: abusemalware
+  type: group
+  description: >
+    Fields for AbuseCH Malware Threat Intel
+  fields:
+  - name: file_type
+    type: keyword
+    description: >
+      File type guessed by URLhaus.
+
+  - name: signature
+    type: keyword
+    description: >
+      Malware familiy.
+
+  - name: urlhaus_download
+    type: keyword
+    description: >
+      Location (URL) where you can download a copy of this file.
+
+  - name: virustotal.result
+    type: keyword
+    description: >
+      AV detection ration.
+
+  - name: virustotal.percent
+    type: float
+    description: >
+      AV detection in percent.
+
+  - name: virustotal.link
+    type: keyword
+    description: >
+      Link to the Virustotal report.
+
+  - name: imphash
+    type: keyword
+    description: >
+      The file's import hash, if available.
+
+  - name: ssdeep
+    type: keyword
+    description: >
+      The file's ssdeep hash, if available.
+
+  - name: tlsh
+    type: keyword
+    description: >
+      The file's tlsh hash, if available.
```

Comment:
```suggestion
```
Can we move `ssdeep` and `tlsh` to `threat.indicator.file.hash.{ssdeep,tlsh}`? I know they're not part of the `hash.*` fieldset, but we can add them and this is where they'd end up.

For `impash`, can we move that to `threat.indicator.file.pe.imphash`? It's part of the PE fieldset now.